tests/integration_tests/resources/dsl/blueprint_parent.yaml

```diff
@@ -8,6 +8,10 @@ inputs:
     type: string
     default: "i"
 
+node_templates:
+  x:
+    type: cloudify.nodes.Root
+
```

Comment:
do you mean we require some instances to exist, to pass the checks at all? I thought no instances at all, would mean that all instance availability checks pass, because of the `all()` calls we use there?

Reply:
That we do, yeah.  It's similar thing in many other places.